docs/presentations/articles/OS_LLMs_Robert.md

```diff
@@ -0,0 +1,278 @@
+<span style="color:red; font-size: 40px; font-weight: bold;">Open-Source LLMs</span>
+
+#### An Overview of Open Source LLMs: What Open Source Alternatives Exist to OpenAI's GPT-3.5 and How Can I Utilize Them?
+
+Autor: Karoly Robert Hegyi
+Datum: 21.12.2023
+
+Author: Karoly Robert Hegyi
+Date: 21.12.2023
+
+Open Source Large Language Models (OSLLMs) are powerful, open-source AI models for natural language processing tasks. Unlike proprietary models, they are transparent, facilitate collaboration, and foster innovation in applications such as translation, text generation, and chatbots.
+
+## Availability of Large Language Models
+
+They are distinguished into:
+
+| Proprietary LLMs                         | Open Source LLMs                          |
+| ---------------------------------------- | ----------------------------------------- |
+| Owned by a company that controls its use | Free of charge and accessible to everyone |
+| License fees required.                   | Free commercial use permitted.            |
+
+Various licenses exist for open-source software, with the majority adhering to the Apache 2.0 standard.
+
+!!! info Apache 2.0
+Allows users to use, distribute, modify, and distribute modified versions of the software for any purpose under the terms of the license, without worrying about license fees.
```

Comment:
To use the [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title)correctly, you need to watch out for the indentation and blank lines, and putting the title in quotes `"`

Also see [this](https://github.com/pkeilbach/htwg-practical-nlp/blob/6bd61b90f6a957234a0c3b4b132ed3bafb0ca63a/docs/lectures/minimum_edit_distance.md?plain=1#L363) example as a reference.